archinstall/lib/user_interaction.py

```diff
@@ -201,8 +195,7 @@ def ask_to_configure_network():
 					break
 				except ValueError:
 					log(
-						"You need to enter a valid gateway (router) IP address.",
-						level=LOG_LEVELS.Warning,
+						'You need to enter a valid gateway (router) IP address',
```

Comment:
Why remove the periods after sentences?

Reply:
In my opinion, this was not necessary for the logs, since it looks somehow strange, but I can leave them for such sentences, where they do not contain ` * ... * `